api/user.json

```diff
@@ -1,7 +1,8 @@
 {
   "login": {
     "code": "A5G98S4K1",
-    "securityQuestion": null
+    "securityQuestion": null,
+    "ageYesNo": null
```

Comment:
Interesting. This is _supposed_ to represent the data coming back from the API. We're using this here as a local session thing because we don't really have an API to worry about. 

Don't change it or anything — just making that observation for later.